site/_scss/blocks/_screenshot.scss

```diff
@@ -0,0 +1,5 @@
+.screenshot {
```

Comment:
I think we should have separate styles for .screenshot and .screenshot--filled

Here's what we do on web.dev

```
.w-screenshot {
  border: 1px solid $GREY_300;
  padding: 4px;

  // This ensures images are scaled to the exact size specified by the
  // width attribute, preventing HiDPI screenshots from being blurry:
  //  <img src="800x500.png" width="400" class="w-screenshot">
  box-sizing: content-box;
  margin-left: -5px;
}

.w-screenshot--filled {
  background-color: $GREY_50;
  padding: 16px;

  // This ensures image size fits to content width with padding(16 * 2)
  // while specifying `box-sizing: content-box;` in .w-screenshot
  max-width: calc(100% - 32px);
  margin-left: 0;
}
```